plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/plugins.php

```diff
@@ -111,6 +110,7 @@ public function test_activate_plugin() {
 		$response       = $this->server->dispatch( $request );
 		$data           = $response->get_data();
 		$active_plugins = Plugins::get_active_plugins();
+		var_dump($data);
```

Comment:
🤭

Reply:
@psealock Thank you for catching it 😄 Removed it in [Remove test code](https://github.com/woocommerce/woocommerce/pull/47316/commits/59d3c2a33164a63088459c2e124ad70f1c76dd79) 🙏